docs/7.x/cluster.md

```diff
@@ -1647,4 +1647,42 @@ Example:
   $ gravity report --since 0s
 ```
 
-This command will collect diagnostics from all Cluster nodes into the specified tarball that you can then submit for evaluation.
+This command will collect diagnostics from all Cluster nodes into the specified tarball that you can then submit for
+evaluation.
+
+If you would like to examine the debug report, here is a quick overview of what information is collected and available
+for examination. First extract the tarball into a directory of your choice. Here we'll extract the tarball into a
+`report` directory with: `mkdir report && tar -zxf report.tar.gz -C report`.
+
+The contents of the report should look similiar to this:
+```bsh
+$ tree report
+report
+├── cluster.json
+├── node-1-debug-logs.tar.gz
+├── node-1-etcd.tar.gz
+├── node-2-debug-logs.tar.gz
+├── node-2-etcd.tar.gz
+├── node-3-debug-logs.tar.gz
+├── node-3-etcd.tar.gz
+├── node-3-k8s-logs.tar.gz
+├── node-3-resources.tar.gz
+├── node-3-status.tar.gz
+└── operation_install.a27a9fa0-c5e6-4a62-973b-2893752c25b6
+```
+
+* `cluster.json` contains the JSON-encoded cluster metadata.
```

Comment:
Nit: I wonder if structuring this information as a table would be a bit cleaner.